evutil.c

```diff
@@ -77,6 +77,7 @@
 #endif
 #include <time.h>
 #include <sys/stat.h>
+#include <net/if.h>
```

Comment:
fails on win32, you can look at the [appveyor ci](https://ci.appveyor.com/project/libevent/libevent/build/job/m4sts7m27yolwhta#L119) if you don't have access to the windows machine

Reply:
[looks like](https://docs.microsoft.com/en-us/windows/win32/api/netioapi/nf-netioapi-if_nametoindex) this is the alternative - `<Iphlpapi.h>`, although not sure, not a win32 fan